GVRf/Framework/jni/engine/importer/assimp_importer.cpp

```diff
@@ -68,4 +68,428 @@ Mesh* AssimpImporter::getMesh(int index) {
 
     return mesh;
 }
+
+bool AssimpImporter::create_instance(JNIEnv *env, const char* class_name,
+        jobject& new_instance) {
```

Comment:
Hi @deepakrawat22 ,

Thank you for going in such details to explain the inclusion of methods in assimp_importer.h/.cpp . 

In assimp_importer_jni.cpp we are parsing the object of GVRAssimpImpoter so that we can call those methods on the same object. Also, all these calls are somehow related to assimp and so our plan for the inclusion of jassimp methods inside assimp_importer. Jassimp is just a bridge for accessing similar functionality on the java side.